components/sup/src/manager/mod.rs

```diff
@@ -823,6 +824,50 @@ impl Manager {
         ))
     }
 
+    pub fn service_group_cfg(
+        mgr: &ManagerState,
+        req: &mut CtlRequest,
+        opts: protocol::ctl::SvcGetCfg,
+    ) -> NetResult<()> {
+        let service_group: ServiceGroup = opts.service_group.ok_or(err_update_client())?.into();
+        let cfg_type = opts
+            .cfg_type
+            .and_then(protocol::types::ServiceCfgType::from_i32)
+            .unwrap_or_default();
+        let mut msg = protocol::types::ServiceCfg {
+            format: Some(protocol::types::service_cfg::Format::Toml as i32),
+            default: None,
```

Comment:
I _think_ that field names can be changed, provided you keep the same tag number, and still retain backwards compatibility of the protocol messages. If that is indeed the case, it would be nice to change this field name from `default` to something more generic (perhaps `config`). 

It may also be useful to add a field indicating which "layer" is being returned... not 100% sure of that right now, though... kind of depends on how we end up wanting to consume this.